### DISCUSSION TITLE
windows打包网页没有标题

### DISCUSSION THREAD

**Discussion by gokele** [tw93/Pake#479] [Q&A]
![image](https://github.com/tw93/Pake/assets/49964567/e0864a0c-a569-4ab8-8671-c7a1a9359ea9)
只有图标

**Comment by Tlntin** [comment]
> ![image](https://github.com/tw93/Pake/assets/49964567/e0864a0c-a569-4ab8-8671-c7a1a9359ea9)
> 只有图标

设定就是没有😆

**Reply by gokele** [reply]
好家伙 有标题多好

**Reply by tw93** [reply]
好勒 我下一个版本加上，正常有标题是什么样子 我看看

**Reply by Tlntin** [reply]
标题的话可以通过配置文件"title"设定（目前设置的应该是空字符串），大概长相就是顶部居中。

**Reply by gokele** [reply]
我用的在线打包方式
